spec/requests/spaces_controller_request_spec.rb

```diff
@@ -67,8 +55,21 @@
       end
     end
   end
+
+  describe "#show" do
+    context "with a branded domain" do
+      let(:space) { create(:space, branded_domain: "beta.example.com") }
+
+      it "redirects to the domain" do
+        get polymorphic_path(space)
+
+        expect(response).to redirect_to "http://beta.example.com"
+      end
+    end
+  end
+
   describe "#destroy" do
-    context "as an Operator using the API" do
+    context "when an API-Operator" do
```

Comment:
Do we now have 2 different types of operators?
`API-Operator` and `Operator`?

`as an Operator using the API` was fine as is!

Reply:
Right now, we have an API key that has `Operator` level privileges. but I don't think this rephrase is useful either.